### PR TITLE
fix(FEC-11451): video tracks are always returned with active false in…

### PR DESCRIPTION
**Pull request by JonathanTGold**
### Description of the Changes

**issue**: video tracks are always returned with active false in native adapter, so when you call kalturaPlayer.getActiveTracks() you always get the video track undefined because it is filtered by active = true

**fix**: add the missing index property in the videoTrack object, 
look at this line https://github.com/kaltura/playkit-js/blob/master/src/player.js#L2381,
Which is the cause of the problem, since the index property does not exist, the condition will always return false

**solves: (FEC-11451)**

### CheckLists

- [ ] changes have been done against master branch, and PR does not conflict
- [ ] new unit / functional tests have been added (whenever applicable)
- [ ] test are passing in local environment
- [ ] Travis tests are passing (or test results are not worse than on master branch :))
- [ ] Docs have been updated

### CODE REVIEW

File: src/engines/html5/media-source/adapters/native-adapter.js [hunk 1]
```diff
@@ -643,13 +644,13 @@ export default class NativeAdapter extends BaseMediaSourceAdapter {
...
   _getParsedAdaptiveVideoTracks(): Array<Track> {
-    //TODO check adaptation in safari hls
     const videoTracks = this._videoElement.videoTracks;
     const parsedTracks = [];
     if (videoTracks) {
       for (let i = 0; i < videoTracks.length; i++) {
         const settings = {
-          //TODO calculate width/height/bandwidth
+          height: this._videoElement.videoHeight,
+          width: this._videoElement.videoWidth,
```
Comment (OrenMe) [inline src/engines/html5/media-source/adapters/native-adapter.js:653]: this value will change on change of video track so we can't know it upfront like this AFAIK

File: src/engines/html5/media-source/adapters/native-adapter.js [hunk 2]
```diff
@@ -544,7 +544,8 @@ export default class NativeAdapter extends BaseMediaSourceAdapter {
           language: '',
           height: videoHeight,
           width: videoWidth,
-          active: true
+          active: true,
+          index: 0
```
Comment (OrenMe) [inline src/engines/html5/media-source/adapters/native-adapter.js:548]: this means we will always say the index is 0 even if track changes

**Review by OrenMe — DISMISSED** [review]

**Review by yairans — DISMISSED** [review]

**Comment by yairans** [comment]
@JonathanTGold is there a test you can update with the new index?

**Review by yairans — APPROVED** [review]

File: src/engines/html5/media-source/adapters/native-adapter.js (hunk 2 above)
Comment (JonathanTGold) [reply src/engines/html5/media-source/adapters/native-adapter.js]: fixed

File: src/engines/html5/media-source/adapters/native-adapter.js (hunk 1 above)
Comment (JonathanTGold) [reply src/engines/html5/media-source/adapters/native-adapter.js]: fixed

**Pull request merged**